staging/src/k8s.io/cri-api/pkg/apis/runtime/v1alpha2/api.proto

```diff
@@ -285,7 +285,30 @@ message LinuxSandboxSecurityContext {
     // * localhost/<full-path-to-profile>: the profile installed on the node.
     //   <full-path-to-profile> is the full path of the profile.
     // Default: "", which is identical with unconfined.
-    string seccomp_profile_path = 7;
+    string seccomp_profile_path = 7 [deprecated=true];
+    // Seccomp profile for the sandbox, defaults to the `RuntimeDefault`
+    // `ProfileType`.
+    SeccompProfile seccomp_profile = 9;
+}
+
+// A seccomp profile which can be used for sandboxes and containers.
+message SeccompProfile {
+    // Available profile types.
+    enum ProfileType {
+        // The container runtime default profile should be used.
+        RuntimeDefault = 0;
+        // Disable seccomp for the sandbox or the container.
+        Unconfined = 1;
+        // A profile defined in a file on the node should be used.
+        Localhost = 2;
+    }
+    // Indicator which `ProfileType` should be applied.
+    ProfileType profile_type = 1;
+    // Indicates a profile defined in a file on the node should be used. The
+    // profile must be preconfigured on the node to work. Must be a descending
+    // path, relative to the kubelet's configured seccomp profile location.
+    // Must only be set if `ProfileType` is `Localhost`.
+    string localhost_profile = 2;
```

Comment:
Should we call this `localhost_profile_path`?

Reply:
Fixed as suggested.